### PR TITLE
Implemented logo & fixed up header

### PR DESCRIPTION
**Pull request by crakila**
Given @matthewhillier's new theme for the site, I think the website has been long overdue a logo. 

I have also fixed up the header slightly and fixed the `<title>` tag to be consistent with the `title` in the `img` tag.

Also, the official FFXIV Eorzea DB links now work properly.

### CODE REVIEW

File: xiv_stats.php [hunk 1]
```diff
@@ -481,9 +486,11 @@ function handleClass($row, $classDef, &$totalArray) {
           <div class="col s12 m6" style="width:100%;">
               <div class="card">
                   <div class="card-content">
-                      <a id="population"><span class="card-title" style="font-size:28pt;">XIVCensus - Character statistics for FFXIV</span></a>
+                      <a id="population"><span class="card-title" style="font-size:28pt;"><img src="img/logo.png" class="logo" title="XIVCensus - Character statistics for FFXIV"/></span></a>                 
+                      <img src="img/lofo.png" />
```
Comment (matthewhillier) [inline xiv_stats.php]: Should that be "logo" rather than "lofo"? 😉

Comment (crakila) [reply xiv_stats.php]: https://github.com/XIVStats/XIVStats/pull/46/commits/5667f1393631499282efdf3bf017b1b263ba3e9d

**Comment by ReidWeb** [comment]
Feel free to merge following Matthew's review - probs best to keep to descriptive commits in future, last commit there is not so.

**Comment by matthewhillier** [comment]
There's something a bit awkward going on with the logos here: 
image

Also, I noticed an issue with one of the new stats I added as the mounts/minions arrays are listed after I tried to access them 🐳. I'll do another PR to fix these.

**Pull request merged**